static/Cost/300_Optimization_Data_Collection/Code/rds_util_template.yaml

```diff
@@ -308,7 +308,8 @@ Resources:
           def lambda_handler(event, context):
               try:
                   for record in event['Records']:
-                      account_id = record["body"]
+                      account_id = account['Id']
```

Comment:
no definition of account in this you need to extract from body

Reply:
added a fix here